.eslintignore

```diff
@@ -0,0 +1,8 @@
+build/
+coverage/
+assets/
+node_modules/
+.grunt/
+
+Gruntfile.js
```

Comment:
I'd prefer not to add the `Gruntfile.js` to `.gitignore` cause it contains the configuration of grunt tasks..

Just a note :)